exchanges/okex/okex.go

```diff
@@ -1153,3 +1176,18 @@ func calculateTradingFee(purchasePrice, amount float64, isMaker bool) (fee float
 func getWithdrawalFee(currency string) float64 {
 	return WithdrawalFees[currency]
 }
+
+// GetBalance returns the full balance accross all wallets
+func (o *OKEX) GetBalance() ([]Balance, error) {
+	var response []Balance
+
+	err := o.SendAuthenticatedHTTPRequest(myWalletInfo, url.Values{}, &response)
+	if err != nil {
+		if common.StringContains(err.Error(), o.ErrorCodes["10004"].Error()) {
```

Comment:
I don't understand this, sorry. If the request fails, then return the object. Why not return the error so engine can be notified the request failed?

Reply:
I think I will change that, I think from memory if you haven't loaded anything in wallet, it just returns request failed.